docs/development/CodingStyle.md

```diff
@@ -257,3 +257,34 @@ It should be based on sound mathematical, physical or computer science principle
 This is important for test code too. Tests shall be based on such principles and real-world properties so they don't just test the current implementation as it happens to be.
 
 Guidelines not tramlines: guidelines are not totally rigid - they can be broken when there is good reason.
+
+#Avoiding Accidental Mistakes
+
+* Always use braces when nesting if/else statements to avoid dangling else (the else branch belongs to the innermost if statement).
+
+```cpp
+if ( var1 ) {
```

Comment:
Only space between keyword and parenhesis:
```
if (var1) {

Reply:
Sure, I can remove it.